wiki/Tournaments/GO/3/fr.md

```diff
@@ -0,0 +1,159 @@
+---
+tags:
+- '#Game_Osu Tourney 3'
+- '#GO 3'
+- '#GO3'
+- Game_Osu Tourney 3
+- GO 3
+- GO3
+---
+# #Game_Osu Tourney 3
+
+![#GO 3 Logo](img/logo.png)
+
+Le **#Game\_Osu Tourney 3** (**#GO 3**) est un tournoi international 1v1 sur osu! sponsorisé par [Logitech](https://www.logitech.com/ "Logitech website") et organisé par ![][flag_FR] [0xy](https://osu.ppy.sh/users/9239673) et ![][flag_FR] [Kaanibaru](https://osu.ppy.sh/users/3117824), supporté par ![][flag_FR] [Sardoche](https://osu.ppy.sh/users/4383702). Il s’agit de sa troisième itération.
```

Comment:
Might want to say "Il s'agit de sa troisième édition" instead of "itération". Itération doesn't mean the edition of something in French so yeah.